python/pyfunc-server/run.sh

```diff
@@ -13,6 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+sigterm_handler() {
+  echo "Terminating.."
+  kill -TERM "$server_pid"
+  wait "$server_pid"
```

Comment:
what happen if we don't wait here?

Reply:
Since the bash is the parent process of the pyfunc server then if it exit immediately it will also kill the pyfunc server.